docs/samples/Microsoft.ML.AutoML.Samples/DataStructures/Movie.cs

```diff
@@ -8,13 +8,19 @@ namespace Microsoft.ML.AutoML.Samples.DataStructures
 {
     public class Movie
     {
-        [LoadColumn(0)]
-        public string UserId;
+        [ColumnName("userId"), LoadColumn(0)]
+        public float UserId { get; set; }
 
-        [LoadColumn(1)]
-        public string MovieId;
 
-        [LoadColumn(2)]
-        public float Rating;
+        [ColumnName("movieId"), LoadColumn(1)]
+        public float MovieId { get; set; }
+
+
+        [ColumnName("rating"), LoadColumn(2)]
+        public float Rating { get; set; }
+
+
+        [ColumnName("timestamp"), LoadColumn(3)]
+        public float Timestamp { get; set; }
```

Comment:
Can this just be removed?

Reply:
It could, but also from in the CodeGeneratorTests you can see that column is added to IgnoredColumnNames collection in ColumnInformation, to make it hidden,

I am working with an existing dataset which already contains timestamp, this sample shows how to ignore it.

The benefit of keeping it this way is at a later time when we add FFM for recommendation, we could show how that column can be used as a user feature.

Reply:
This is potentially another use case for having DataFrame; If we had DataFrame, we could have loaded the dataset as IDataView without needing it to match the Movie class in the statement below:
```
IDataView trainDataView = mlContext.Data.LoadFromTextFile<Movie>(TrainDataPath, hasHeader: true, separatorChar: ',');
```
Then the Movie class wouldn't have needed Timestamp field.

cc: @pgovind